module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,60 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <utility>
+#include <vector>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressedData;
+
+    for (const auto& row : bitmap) {
+        if (row.empty())
+            continue;
+
+        uint8_t prevElem = row[0];
+        int cnt = 0;
```

Comment:
Why count is `int`? in fact it is guaranteed that it will not exceed uint_8 (even by pair type)